pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -748,8 +748,9 @@ self: super:
     ];
     propagatedBuildInputs = [ self.xauth ]
                          ++ lib.optionals isDarwin [ self.libX11 self.xorgproto ];
-    prePatch = ''
-      sed -i 's|^defaultserverargs="|&-logfile \"$HOME/.xorg.log\"|p' startx.cpp
+    # Link to system's xserverrc
+    fixupPhase = ''
+      ln -sf /etc/X11/xinit/xserverrc $out/etc/X11/xinit/xserverrc
```

Comment:
we should never override fixupPhase, there's important things that happen in it.
Please use `postFixup`.

Reply:
Thanks!